pkg/log/logger.go

```diff
@@ -294,12 +294,14 @@ func parseLevel(level string) *zapcore.Level {
 	// parse numeric
 	levelNum, err := strconv.ParseInt(level, 10, 31)
```

Comment:
zap's level is a `int8` we should adjust ParseInt to return something that fits in that value. Leave the rest of the code as is.
We also need to extend the TestParseLevel unit tests to make sure we catch these outliers

Reply:
Sure, updated with adding more test cases